Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG PYTHON_V=3.8
+
+FROM python:${PYTHON_V}-slim AS base
+
+WORKDIR /app
+
+COPY . .
+
+
+FROM python:${PYTHON_V}-slim AS run
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY --from=base /app /app
+
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt && \
```

Comment:
Make sure that the `requirements.txt` file is present in the context directory. Otherwise, the `pip install -r requirements.txt` command will fail.